js_modules/dagster-ui/packages/ui-core/src/automation/VirtualizedAutomationSensorRow.tsx

```diff
@@ -196,6 +228,16 @@ export const VirtualizedAutomationSensorRow = forwardRef(
             )}
           </RowCell>
         </AutomationRowGrid>
+        <SensorDryRunDialog
+          isOpen={showTestTickDialog}
+          onClose={() => {
+            setShowTestTickDialog(false);
+          }}
+          currentCursor={cursor || ''}
+          name={sensorData?.name || ''}
+          repoAddress={repoAddress}
+          jobName={sensorData?.targets?.[0]?.pipelineName || ''}
+        />
```

Comment:
One issue we've had in the past is that these pages auto-refresh, and sometimes the auto-refresh causes the table to re-render it's rows, resetting things like `showTestTickDialog` if we're not careful.

I don't think that is happening anymore in most places, but could you try opening this dialog and then letting the page sit there for ~30s to verify that the dialog doesn't close on it's own when the list of sensors refreshes?

Let me know if it happens and we can look at a fix :-)

Reply:
tested with dailog open, doesn't close on its own